tests/foreman/api/test_contentview.py

```diff
@@ -575,24 +520,19 @@ def test_positive_promote_1(self):
         @Feature: ContentView
 
         """
-        content_view = entities.ContentView(organization=self.org.id)
-        content_view.id = content_view.create_json()['id']
+        content_view = entities.ContentView(organization=self.org.id).create()
         content_view.publish()
+        content_view = content_view.read()
 
         # Promote the content view version several times.
-        cvv = entities.ContentViewVersion(
-            id=content_view.read_json()['versions'][0]['id']  # only one ver
-        )
         for _ in range(REPEAT):
-            lc_env_id = entities.LifecycleEnvironment(
-                organization=self.org.id
-            ).create_json()['id']
-            cvv.promote(lc_env_id)
+            lce = entities.LifecycleEnvironment(organization=self.org).create()
+            content_view.version[0].promote(lce.id)
 
         # Does it show up in the correct number of lifecycle environments?
         self.assertEqual(
-            len(content_view.read_json()['versions'][0]['environment_ids']),
-            REPEAT + 1
+            REPEAT + 1,
+            len(content_view.read().version[0].read().environment),
```

Comment:
Ah, I was just sorting them with vim's built-in sorter out of habit. I'm happy to revert. There's no great reason to prefer one version over the other.